packages/y-indexeddb/src/index.ts

```diff
@@ -274,8 +274,8 @@ export const createIndexedDBProvider = (
       doc.off('update', handleUpdate);
       doc.off('destroy', handleDestroy);
     },
-    cleanup() {
-      // todo
+    async cleanup() {
+      (await dbPromise).delete('workspace', id);
```

Comment:
There might have some edge case.

For example, it should throw an error when the provider is still connecting.